loops.js

```diff
@@ -49,4 +57,12 @@
 // >> 7
 // >> 9
 // ==========================================
-
+for ( i = 0; i <= 9; i++ ) {
+    if (i < 10 && i > 5) {
+        console.log(">> " + i);
+    } else if (i < 6 && i > 2) {
+        console.log("> " + i);
+    } else {
+        console.log(i);
+    }
+}
```

Comment:
Ook iets een andere benadering dan in de uitwerkingen, maar met dezelfde uitkomst. Vele wegen leiden naar Rome!